lib/src/web_driver.dart

```diff
@@ -141,9 +141,18 @@ class WebDriver implements SearchContext {
   Mouse get mouse => new Mouse._(this);
 
   /// Take a screenshot of the current page as PNG.
+  Future<String> captureScreenshotAsBase64() async =>
+      await getRequest('screenshot');
+
+  /// Take a screenshot of the current page as PNG.
+  Future<List<int>> captureScreenshotAsList() async {
+    var base64Encoded = captureScreenshotAsBase64();
+    return BASE64.decode(await base64Encoded);
+  }
+  /// Take a screenshot of the current page as PNG.
+
   Stream<int> captureScreenshot() async* {
```

Comment:
What do you think about marking this method as deprecated (with a notice to redirect people to the new captureScreenshot as list)? As far as I can tell this method offers only disadvantages and no advantages. We should remove it in a future version of WebDriver.